assets.json

```diff
@@ -3,570 +3,476 @@
       {
         "name": "Moonriver",
```

Comment:
Hey as for the name. I would indeed add something that references that this token is Composable Finance Moonriver IBC for example